ExpenseTracker/ExpenseTracker/Views/Wallets/Delete.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@model ExpenseTracker.Application.ViewModels.Wallet.WalletViewModel
+
+@{
+    ViewData["Title"] = "Delete";
+}
+
+<h3>Are you sure you want to delete this?</h3>
```

Comment:
maybe it would be better to give _wallet name_ instead of "this"

Reply:
This is shown below